indico/modules/events/abstracts/__init__.py

```diff
@@ -134,6 +134,11 @@ def _reviewing_area_visible(conf):
             return False
         return bool(get_user_tracks(conf.as_event, session.user))
 
+    def _submit_abstract_visible(conf):
+        if not session.user or not conf.has_feature('abstracts'):
```

Comment:
OTOH... We should give enough information in the general CFA page about the possibility of submitting abstracts, so maybe this logic is fine.